draft-ietf-quic-http.md

```diff
@@ -942,6 +942,11 @@ A server MAY interleave one or more PUSH_PROMISE frames (see
 PUSH_PROMISE frames are not part of the response; see {{server-push}} for more
 details.
 
+The HEADERS and PUSH_PROMISE frames might reference concurrent updates to the
+QPACK table.  While these updates are not directly part of the message exchange,
```

Comment:
I agree with Martin that we should say 'QPACK dynamic table' here.  

I'm not crazy about 'concurrent'.  What if we just said 'might reference updates'?